File: benchmarks/boolean/sphere.lua
options = gmshOptions()
options:initOptions()
options:numberSet('Mesh', 0, 'CharacteristicLengthFactor', 0.9grep -IIr)

R = 1.0;
myModel = GModel();
myModel:addSphere(0,0,0,R);

myModel:setAsCurrent();

myModel:mesh(2);

File: benchmarks/boolean/sphere.py
from dgpy import *
from gmshpy import *

GmshSetOption('Mesh', 'CharacteristicLengthFactor', 0.9)

R = 0.3;
myModel = GModel();
myModel.addSphere(0.5,0.5,0.5,R);

myModel.setAsCurrent();

myModel.mesh(2);
myModel.save("sphere.stl");
